app/ledger.js

```diff
@@ -838,7 +838,10 @@ var run = (delayTime) => {
     delayTime = client.timeUntilReconcile()
     if (delayTime === false) delayTime = 0
   }
-  if (delayTime > 0) return setTimeout(() => { if (client.sync(callback) === true) return run(0) }, delayTime)
+  if (delayTime > 0) {
+    active = client
```

Comment:
i don't think `active` is necessary. you can just check `if (client && client.sync) ...`